test/rrd/Makefile

```diff
@@ -0,0 +1,59 @@
+# User-customizable variables:
```

Comment:
Ideally, this would be cmake along with a clone of the Relacy source code in one step. Leaving this here for now, as I haven't chosen to integrate the build into CI (some rationale in the README).